src/components/ServerList.tsx

```
@@ -126,1062 +109,896 @@ const themeLightWarm = themeQuartz.withPart(colorSchemeLightWarm);
-			.fetchDataAndFilter()
-			.then(() => {
-				serverList.moveListDown();
-				let stringList: Array<{ server: string; motd: string }> = [];
-				let obj: any = {};
+  useEffectOnce(() => {
+    setRandomText(getRandomText());
+    serverList
+      .fetchDataAndFilter()
+      .then(() => {
+        serverList.moveListDown();
+        let stringList: Array<{ server: string; motd: string }> = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'stringList' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)